TYPO3.Media/Classes/TYPO3/Media/Domain/Repository/AssetRepository.php

```diff
@@ -151,11 +156,20 @@ public function findUntagged(AssetCollection $assetCollection = null)
      */
     public function countUntagged(AssetCollection $assetCollection = null)
     {
-        $query = $this->createQuery();
-        $query->matching($query->isEmpty('tags'));
-        $this->addImageVariantFilterClause($query);
```

Comment:
no, the methods `addImageVariantFilterClause` and `addAssetCollectionToQueryConstraints`